public/app/themes/justice/error-pages/maintenance.html

```diff
@@ -70,7 +34,7 @@ <h1 class="o-title o-title--page">Website temporarily unavailable</h1>
     <div class="l-primary"></div>
     <div class="l-secondary c-copyright">
       <div class="c-copyright-notice">
-        <img src="/app/themes/clarity/dist/images/crown_copyright_logo.png" alt="crown copyright logo">
+        <img src="/app/themes/justice/src/img/crown_copyright_logo.png" alt="crown copyright logo">
```

Comment:
Hi @EmilyHazlehurst the stacks will need to reference `dist` - `src` doesn't exist in the file system.